Dockerfile

```diff
@@ -19,14 +19,20 @@ COPY main.go main.go
 COPY controllers/ controllers/
 
 # build without giving the arch, so that it gets it from the machine
-RUN CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -a -o image-reflector-controller main.go
+RUN CGO_ENABLED=0 go build -a -o image-reflector-controller main.go
 
 FROM alpine:3.12
 
+LABEL org.opencontainers.image.source="https://github.com/fluxcd/image-reflector-controller"
+
 RUN apk add --no-cache ca-certificates tini
 
 COPY --from=builder /workspace/image-reflector-controller /usr/local/bin/
 
+# Create minimal nsswitch.conf file to prioritize the usage of /etc/hosts over DNS queries.
+# https://github.com/gliderlabs/docker-alpine/issues/367#issuecomment-354316460
+RUN [ ! -e /etc/nsswitch.conf ] && echo 'hosts: files dns' > /etc/nsswitch.conf
```

Comment:
Would this have a bit better caching behaviour if it were above the COPY directive?

Reply:
Good point! I just copied it over as seen in other components' Dockerfiles. I will adjust, can probably be the first directive considering it's static nature.